modules/gke-standard-private-cluster/variables.tf

```diff
@@ -127,3 +127,9 @@ variable "gpu_pools" {
 variable "tpu_pools" {
   type = list(map(any))
 }
+
+variable "datapath_provider" {
+  description = "Default to Dataplanev2"
+  type        = string
+  default     = "ADVANCED_DATAPATH"
```

Comment:
what is the other options? maybe also update the read me in rag or ray?

Reply:
How are we autogenerating the READMEs for modules? I'm happy to regenerate that. What would you suggest to add into the higher level READMEs? This is just setting a default, lower level option defined in the GKE cluster modules:

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#datapath_provider

Reply:
manual generation